.envrc

```diff
@@ -36,7 +36,9 @@ fi
 
 # Set up env vars and build inputs from flake.nix automatically for nix users.
 # If you don't use nix, you can safely ignore this.
-if command -v nix &> /dev/null
+# You can set the DISABLE_NIX environment variable if you're in an environment
+# where nix is pre-installed (e.g. gitpod) but you don't want to use it.
+if command -v nix &> /dev/null && [ -z ${DISABLE_NIX+x} ]
```

Comment:
for @janpio 😄

Reply:
Yay, updated on my Gitpod config: 
<img width="401" alt="image" src="https://github.com/prisma/prisma-engines/assets/183673/d67ccc30-4139-41e2-8641-e3c0c7b9d766">